server/server.js

```diff
@@ -7,14 +7,13 @@ const db = require('../server/db/db-connection.js');
 const REACT_BUILD_DIR = path.join(__dirname, '..', 'client', 'build');
 const app = express();
 
-
 const config = {
     authRequired: false,
     auth0Logout: true,
-    secret: process.env.SECRET,
-    baseURL: process.env.BASEURL,
-    clientID: process.env.CLIENTID,
-    issuerBaseURL: process.env.ISSUERBASEURL
+    secret: process.env.AUTH0_SECRET,
```

Comment:
A nit, but variable naming is important. You will end up with more than one `secret` for your app, so you want to give each secret a clear name!